event/src/main/scala/com/improving/app/event/domain/Event.scala

```diff
@@ -139,34 +132,57 @@ object Event extends StrictLogging {
                   x match {
                     case ScheduledEventState(info, meta) =>
                       command.request match {
-                        case startEventCommand: StartEvent => startEvent(info, meta, startEventCommand)
-                        case delayEventCommand: DelayEvent => delayEvent(Right(info), meta, delayEventCommand)
+                        case startEventCommand: StartEvent =>
+                          if (Instant.now().getEpochSecond > info.getExpectedStart.seconds)
```

Comment:
maybe should be `>=`?

Reply:
Maybe a little slop even on the early side would be nice, to allow an event to start a few minutes early if the user requests it